oh_staff_ui/management/commands/import_names.py

```diff
@@ -0,0 +1,66 @@
+from django.core.management.base import BaseCommand
+from oh_staff_ui.models import (
+    ProjectItem,
+    Name,
+    ItemNameUsage,
+    AuthoritySource,
+    NameType,
+)
+from csv import DictReader
+import os
+
+
+class Command(BaseCommand):
+    help = "Imports Name data from a CSV file"
+
+    def add_arguments(self, parser):
+        parser.add_argument("filename", type=str)
+
+    def handle(self, *args, **options):
+        # open and read input CSV - assuming it's in the fixtures folder
+        filepath = os.path.join("oh_staff_ui/fixtures", options["filename"])
+        with open(filepath, mode="r") as f:
+            dict_reader = DictReader(f)
+            name_dicts = list(dict_reader)
+
+        print(f"Found {len(name_dicts)} rows of name usage to import.")
+        for row in name_dicts:
+            item = ProjectItem.objects.get(ark=row["ARK"])
+            source = self.get_or_create_source(row["SOURCE"])
+            name = self.get_or_create_name(source, row["VALUE"])
+            type = self.get_or_create_nametype(row["TYPE"])
+            # avoid duplicates - only add ItemNameUsages that don't exist yet
+            if not ItemNameUsage.objects.filter(
+                item=item, name=name, type=type
+            ).exists():
+                usage = ItemNameUsage(item=item, name=name, type=type)
+                usage.save()
+        print("Finished importing names associated with items.")
+        total_names = Name.objects.filter().count()
```

Comment:
FYI, counting all objects is better done with `.objects.count()`, since no filtering is being done.  You can also use `.objects.all().count()`, but the `all()` or `filter()` isn't needed.  This is OK here, just something to watch in future scripts.